neurodamus/commands.py

```diff
@@ -67,6 +67,9 @@ def neurodamus(args=None):
         --num-target-ranks=<number>  Number of ranks to target for dry-run load balancing
         --coreneuron-direct-mode     Run CoreNeuron in direct memory mode transfered from Neuron,
                                      without writing model data to disk.
+        --report-buffer-size=<number> Override the size in MB each rank will allocate for each 
```

Comment:
where is the current default being set?  Could that be built into the logic of the command line, and also in the validation?  That way whoever has to chase it down in the future will find it more easily.

Reply:
It's a good point that of the several 'default' values in the code, it is not clear which is the one actually used. I'll make another ticket to fix this with some info on my recent code browsing. What I would like is that libsonatareport makes its default accessible to outsize applications so they can query it

Reply:
>  What I would like is that libsonatareport makes its default accessible to outsize applications so they can query it

I'm not sure that's necessary; we can just have a default in ND, and always set it to that unless the user requests a different value - basically ignore that that libsonatareport has a default.

Reply:
why does libsonatareport have a default? It should be set by coreneuron/neurodamus no?